### ISSUE TITLE
Determine plan for next steps for the Knowledge Layer Connector

### ISSUE THREAD

**Issue by chrizmc** [COVESA/cdsp#44]
Issue #21 concluded the initial consideration of Why/What for the Knowledge Layer Connector. Now we need to discuss and plan for the next steps in its implementation

Please find here the results:

![Image](https://github.com/COVESA/cdsp/assets/36862281/b9d2ebbb-f084-4dce-b366-fabe870bbe3d)
![Image](https://github.com/COVESA/cdsp/assets/36862281/204d79ed-3ee6-4b42-95b5-1fbcd8937615)
![Image](https://github.com/COVESA/cdsp/assets/36862281/0952e9c1-8194-4b0b-bd47-81a0022fad00)
![Image](https://github.com/COVESA/cdsp/assets/36862281/fdfd6128-2a95-4998-8aca-62d263b65353)

**Comment by chrizmc** [comment]
we are preparing first Pull Request which contains basic folder structure and basic content of readme file

**Comment by slawr** [comment]
Hi @chrizmc in last weeks call you mentioned that @claireqiu and yourself had met and had a plan for KL up towards the Spring AMM. As we discussed in the call I would like to get the high level tasks to complete that plan in this Kanban as its part of the project. With that in mind can you create new tickets as needed to represent the plan and we can close this ticket as done as it's 'determine plan' not 'execute plan'? If you have a different idea please discuss.

It doesn't necessarily have to extreme detail its more about people outside BMW understanding the plan.

**Comment by claireqiu** [comment]
<img width="1115" alt="Image" src="https://github.com/user-attachments/assets/5b23d95b-15a9-4f43-8000-3dd4f610306e" />

**Comment by chrizmc** [comment]
@slawr I agree, this is completed with [https://github.com/COVESA/cdsp/pull/50](https://github.com/COVESA/cdsp/pull/50) . Remaining open tasks are handled with [https://github.com/COVESA/cdsp/issues/77](https://github.com/COVESA/cdsp/issues/77)
